earth_enterprise/rpms/opengee-fusion/pre-uninstall.sh

```diff
@@ -14,12 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CHKCONFIG="/sbin/chkconfig"
+
 set +x
 
 #-----------------------------------------------------------------
 # Main Function
 #-----------------------------------------------------------------
 
-/etc/init.d/gefusion stop
+service gefusion stop
+test -f $CHKCONFIG && $CHKCONFIG --del gefusion
```

Comment:
That's not a good way to write this test. You'll end up exiting with a failure error code, if `gefusion` is not registered as a service.

Reply:
If it is not registered, that maybe is a failure condition...at least it is presumed that it should have been, since it is registered in the post-install, and this is logic we have used before.

Reply:
Whether we've used this logic before, or if it's a failure condition, I think we may still want not to fail in a pre-uninstall script, unless there's a really serious condition that can cause data loss, or some other condition that's hard to recover from.

A user should usually not unregister services manually, but there really ins't anything preventing that from happening, and I don't think that's a good enough reason to prevent a package from being able to uninstall.